build_script.py

```diff
@@ -373,6 +380,12 @@ def main(args=sys.argv[1:]):
         help="Path to swift-corelibs-foundation build products, which the "
              "tests will be linked against.",
         required=strategy.requires_foundation_build_dir())
+    test_parser.add_argument(
+        "--foundation-install-prefix",
+        help="Path to the installation location for swift-corelibs-foundation "
+             "build products; CoreFoundation dependencies are expected to be "
+             "found under $foundation_build_dir/$foundation_install_prefix.",
```

Comment:
Same as above, I think it'd be nice to include what the `%(default)s` value is in this description.